### PR TITLE
misc corrections

### PR DESCRIPTION
**Pull request by c2nelson**
various minor edits that needed to be made.  i removed the outdated what's new section, modified some absolute links to relative links and a few other little things that were irking me.

### CODE REVIEW

File: _pages/_resources/disability-rights-guide.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Complaints of title II violations may be filed with the Department of Justice wi
 >Washington, D.C. 20530
 >{{site.contact.tollfree}} (voice)
 >{{site.contact.tty}} (TTY)
->[ADA.gov](https://www.ada.gov)
+>[ADA.gov]({{'/index' | relative_url}})
```
Comment (kewitham) [inline _pages/_resources/disability-rights-guide.md:60]: I think we want these to be `{{ '/' | relative_url }}` rather than using /index

**Review by kewitham — CHANGES_REQUESTED** [review]

File: _pages/_resources/disability-rights-guide.md (hunk 1 above)
Comment (c2nelson) [reply _pages/_resources/disability-rights-guide.md]: thank you, @kewitham.  changes made.

**Review by kewitham — APPROVED** [review]
lgtm!

**Review by vwilliamsn — APPROVED** [review]
Thanks @c2nelson!

**Review by okkays — APPROVED** [review]

**Pull request merged**